platform/bootstrap/userdata-aws

```diff
@@ -10,6 +10,7 @@ DRAIN_MANAGER=${DRAIN_MANAGER:-false}
 OVERLAY_NETWORKS=${OVERLAY_NETWORKS:-ampnet}
 #MIRROR_REGISTRIES=
 #DOCKER_DEVICE=/dev/sdn
+SYSTEMD_DOCKER_OVERRIDE=/etc/systemd/system/docker.service.d/docker.conf
 
 _install_docker(){
   wget -qO- https://get.docker.com/ | sh || return 1
```

Comment:
Just a note for now, but we're probably going to want to ensure we install specific Docker versions pretty soon.